.gitignore

```diff
@@ -1,3 +1,4 @@
-vendor/
+/vendor
+/bin
+/tests-coverage
```

Comment:
the best practice (IMHO) is to add `/` at the end of ignored directories (just to know that this is a directory...)